GISSR_FINAL CHALLENGE_REFILOE SEKALELI.Rmd

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Final Challenge Instructions"
+title: "Exploring the Relationship Between Arable Land Use, Food Insecurity, and Healthcare Facilities in Mafeteng"
 author: "REFILOE SEKALELI"
 date: "`r Sys.Date()`"
 output: 
   html_document:
-    theme: flatly
+    theme: cayman
```

Comment:
Had to change this to a default theme to knit the .html file on my end (didn't know which library to install to include the cayman theme)